back-end/src/services/issue.js

```diff
@@ -72,9 +72,28 @@ async function updateIssue( issue ) {
   return {...msg, ...updateIssue };
 }
 
+async function completedIssue() {
+  const msg = {
+    select: true,
+    info: 'successful'
+  }
+
+  let completedIssue = await issueModel.completedIssue();
+
+  if( !completedIssue ) {
+    msg.select = false;
+    msg.info = 'is not possible select';
```

Comment:
a mensagem ficou um pouco estranha, o segundo verbo nesse caso precisa do auxiliar
`is not possible to select`